apps/common/translations/src/en/buttons.json

```diff
@@ -0,0 +1,3 @@
+{
+  "start-home": "Start Now"
+}
```

Comment:
Is this button in the accessibility page? If not I don't think we need it

Reply:
Removed